test/jdk/java/util/zip/Zip64Offsets.java

```diff
@@ -0,0 +1,109 @@
+/*
```

Comment:
Hello Liam, given that this PR is addressing an issue in the `java` launcher, I think this test should reside in `test/jdk/tools/launcher/` directory.

Reply:
I moved it to `test/jdk/tools/launcher/`, thanks!

(I modeled this after some other zip64 tests, which I now realize were originally added as tests for `java.util.zip`.)